# This patch is done because there is no way to enable SSO/SAML at the moment

diff --git a/internal/api/api.go b/internal/api/api.go
index d9016fd0..924ce018 100644
--- a/internal/api/api.go
+++ b/internal/api/api.go
@@ -175,7 +175,7 @@ func NewAPIWithVersion(ctx context.Context, globalConfig *conf.GlobalConfigurati
 		})

 		r.Route("/sso", func(r *router) {
-			r.Use(api.requireSAMLEnabled)
+			// r.Use(api.requireSAMLEnabled)
 			r.With(api.limitHandler(
 				// Allow requests at the specified rate per 5 minutes.
 				tollbooth.NewLimiter(api.config.RateLimitSso/(60*5), &limiter.ExpirableOptions{
